### DISCUSSION TITLE
System Screen Capture on by default on startup

### DISCUSSION THREAD

**Discussion by marekburiak** [awawa-dev/HyperHDR#808] [Q&A] [answered]
Hi, how do I turn on "System Screen Capture" automatically when HyperHDR starts? I know I can turn it on in Remote Control during runtime but how do I make that change permanent?

**Comment by awawa-dev** [comment, marked as answer]
Hi
In video capturing tab from the main menu there is instance control next to the system grabber and you must enable it.
